io.github.xyproto.zsnes.yaml

```diff
@@ -0,0 +1,76 @@
+id: io.github.xyproto.zsnes
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: zsnes
+finish-args:
+  - --share=ipc
+  - --device=all
+  - --socket=x11
+  - --allow=multiarch
+  - --socket=pulseaudio
+  - --filesystem=host:rw
```

Comment:
this remain a very broad permission

Reply:
It#s the only Permission that makes it fully work. And is it not more harmful than `home`.